### ISSUE TITLE
[BUG] Crash on version 1.0.58

### ISSUE THREAD

**Issue by darthmooguy** [aristocratos/bpytop#238]
**Describe the bug**

The app crashes on startup.

**To Reproduce**

Run `bpytop` in a terminal.

**Expected behavior**

The app starts normally. It worked with prior versions.

**Info (please complete the following information):**
 - bpytop version: 1.0.58 (installed from pip3, same behavior when installed from Fedora repositories)
 - psutil version: 5.7.2
 - (Linux) Linux distribution and version: Fedora 33
 - Terminal used: Tilix (libvte based)
 - Font used: Monospace Regular
 - Python version: Python 3.9.1

**Additional context**

contents of `~/.config/bpytop/error.log`

```
11/01/21 (15:08:44) | ERROR: Data collection thread failed with exception: type object 'MemCollector' has no attribute 'disks'
Traceback (most recent call last):
  File "/home/alex/.local/lib/python3.9/site-packages/bpytop.py", line 2824, in _runner
    collector._draw()
  File "/home/alex/.local/lib/python3.9/site-packages/bpytop.py", line 3346, in _draw
    MemBox._draw_fg()
  File "/home/alex/.local/lib/python3.9/site-packages/bpytop.py", line 2056, in _draw_fg
    for name in mem.disks.keys():
AttributeError: type object 'MemCollector' has no attribute 'disks'
11/01/21 (15:08:44) | WARNING: Exiting with errorcode (1). Runtime 0:00:02 
```

**Comment by aristocratos** [comment]
@darthmooguy 
Should be fixed in v1.0.59

**Comment by darthmooguy** [comment]
I confirm, it works now!

Thanks for the quick update!
